README.md

```diff
@@ -1,3 +1,3 @@
 # TEST01
-Testing git and github to practice version control and keep files of assigments and projects.
+Testing Git and Github to practice version control and keep files of assigments and projects.
```

Comment:
wtf are you doing kids? too much free time? go help your mom filling water bottle.

Reply:
LGTM btw